packages/ui/SurveysList/actions.ts

```diff
@@ -29,251 +26,54 @@ export const getSurveyAction = async (surveyId: string) => {
   return await getSurvey(surveyId);
 };
 
-export const duplicateSurveyAction = async (environmentId: string, surveyId: string) => {
+export const copySurveyToOtherEnvironmentAction = async (
+  environmentId: string,
+  surveyId: string,
+  targetEnvironmentId: string,
+  targetProductId: string
```

Comment:
Do we need the targetProductId here or can we infer this from the targetEnvironmentId?
Otherwise what would happen if they don't match?